tests/integration/test_api.py

```diff
@@ -381,3 +387,82 @@ def test_annotation_event(client, monkeypatch, httpserver):
             },
         )
     assert result.status_code == 200
+
+
+def test_prediction_collection_no_filter(client):
+    result = client.simulate_get("/api/v1/predictions/")
+    assert result.status_code == 200
+    assert result.json == {"count": 0, "predictions": [], "status": "no_predictions"}
```

Comment:
As this part of the test is independant from the rest, it's better to separate it.

```suggestion
def test_prediction_collection_no_result(client):
    result = client.simulate_get("/api/v1/predictions/")
    assert result.status_code == 200
    assert result.json == {"count": 0, "predictions": [], "status": "no_predictions"}

def test_prediction_collection_no_filter(client):
```